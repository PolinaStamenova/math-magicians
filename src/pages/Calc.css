.calc-holder {
  display: flex;
  justify-content: space-between;
  padding: 0 5rem;
  margin-top: 3rem;
}

.calculator {
  width: 500px;
}
