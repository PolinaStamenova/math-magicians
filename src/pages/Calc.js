/* eslint-disable react/prefer-stateless-function, quotes */

import React from "react";
import Calculator from "../components/Calculator";
import "./Calc.css";

function Calc() {
  return (
    <div className="calc-holder">
      <h2 className="sub-title">Let`s do some math!</h2>
      <Calculator />
    </div>
  );
}

export default Calc;
