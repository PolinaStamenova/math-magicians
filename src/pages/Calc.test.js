import React from "react";
import renderer from "react-test-renderer";
import Calc from "./Calc";

it("expect calculator page to match working snapshot", () => {
  const calculatorPage = renderer.create(<Calc />).toJSON();
  expect(calculatorPage).toMatchSnapshot();
});
