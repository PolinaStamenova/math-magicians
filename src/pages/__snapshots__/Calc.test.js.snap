// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect calculator page to match working snapshot 1`] = `
<div
  className="calc-holder"
>
  <h2
    className="sub-title"
  >
    Let\`s do some math!
  </h2>
  <div
    className="calculator"
  >
    <p
      className="calculator-result"
    >
      0
    </p>
    <div
      className="calculator-buttons"
    >
      <button
        className="grey"
        name="AC"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="grey"
        name="+/-"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="grey"
        name=""
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="orange"
        name="รท"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="grey"
        name="7"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="grey"
        name="8"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="grey"
        name="9"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="orange"
        name="x"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="grey"
        name="4"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="grey"
        name="5"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="grey"
        name="6"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="orange"
        name="-"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="grey"
        name="1"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="grey"
        name="2"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="grey"
        name="3"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="orange"
        name="+"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="grey, zero"
        name="0"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="grey"
        name="."
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="orange"
        name="="
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
